server.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+var express = require('express');
+var bodyparser = require('body-parser');
+var Adjective = require('./lib/adjective.js');
```

Comment:
good organization here. Convention is to do the npm requires first, then any of your own files.